ffsend.py

```diff
@@ -617,7 +618,9 @@ def do_set_params():
         # Assume they tried to upload a nonexistent file
         raise OSError("File %s does not exist" % args.target)
 
+def cli():
```

Comment:
To avoid the extra function, we could also do
```python
def main(args=None):
    if args is None: 
          args = sys.argv[1:]
    ...
```

which might be clearer.